kubernetes/services/signing-service.yml.j2

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: signing-service
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: signing-service
+    spec:
+      containers:
+        - name:             signing-service
+          image:            {{ docker_registry }}/{{image_prefix }}signing-service:{{ image_version }}
+          imagePullPolicy:  Always
+          args:             [
+              "{{ cluster_address }}",
+              "5",
+              "--concent-cluster-port", "9055"
+          ]#,"--sentry-dsn-from-env"]
+          ports:
+            - containerPort: 9055
+              name: external-port
+          env:
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: sentry-data-source
+                  key:  sentry_data_source_name
+          resources:
+            requests:
+              memory: "0.3Gi"
+              cpu:    "50m"
```

Comment:
Let's give it less memory. Preferably 100 MB or less.

Reply:
Done.